VoteStakeRegistry/components/Account/Account.tsx

```diff
@@ -49,19 +50,29 @@ const NotificationsPlugin = () => {
   return null
 }
 
+const DelegatePlugin = () => {
```

Comment:
nit: the delegation logic is not a plugin

Reply:
removed function